content/en/platform/corda/1.6/cenm/config-angel-service-parameters.md

```diff
@@ -0,0 +1,92 @@
+---
+aliases:
+- /config-angel-service-parameters.html
+date: '2024-09-24'
+menu:
+  cenm-1-6:
+    identifier: cenm-1-6-config-angel-service-parameters
+    parent: cenm-1-6-configuration
+    weight: 255
+tags:
+- config
+- angel service
+title: Angel Service Configuration Parameters
+---
+
+# Angel Service Configuration Parameters
+
+The configuration references for the Angel Service are given below:
+
+* **pollingTimeSeconds**:
+*(Optional)* The time in seconds to wait before polling the Zone Service.
+
+* **networkParametersFile**:
+The path to the network parameters file. This is the plain-text version of the network parameters also used when setting the initial network parameters, this is not the binary `network-parameters` file that Corda nodes use. Only used for the `NETWORK_MAP` service type.
+
+  {{< important >}}
+  The `networkParametersFile` property can only be specified if `service.type` is set to `NETWORK_MAP`.
+  {{< /important >}}
+
+* **service**:
+
+  * **type**:
+  The main class of the plugin being loaded.
+
+  * **jarFile**:
+  *(Optional - defaults to one of: `identitymanager.jar`, `networkmap.jar`, `signer.jar`)* The path to the service JAR.
+
+  * **enmListener**:
+  Information about the way the service will communicate with the rest of the CENM deployment.
+
+    * **host**:
+    *(Optional)* The host or IP address of the service.
+
+    * **port**:
+    The port that the service binds to, and other CENM components it connects to.
+
+    * **verbose**:
+    *(Optional)* Enables verbose logging for the socket layer.
+
+    * **reconnect**:
+    Determines if a client should attempt to reconnect if the connection is dropped.
+
+    * **ssl**:
+    See [SSL Settings]({{< relref "../../../../../en/platform/corda/1.6/cenm/config-ssl.md" >}}).
+
+  * **pluginJar**:
+  *(Optional)* The absolute path to the JAR file of the workflow plugin.
+
+  * **networkRootTrustStore**:
+  Information about the network root trust store file.
+
+    * **location**:
+    The path to the network parameters file. Only used for the Network Map Service.
+
+    * **password**:
+    The password for the truststore file.
+
+    * **rootAlias**:
+    The root alias.
+
+    * **validate**:
+    *(Optional)* Validate the network truststore path.
```

Comment:
@tomstark99 Is this Boolean?

Reply:
Hi @paulmoloneyr3 yeah, **but** I just checked the other config pages and although all truststore config can use the validate bool, (because they all use the same config class) its not mentioned anywhere, so maybe we just remove it from here (and anywhere else I have mentioned it)